README.md

```diff
@@ -6,22 +6,43 @@
 2. Describe why a virtual DOM might be useful
 3. Describe some of the tradeoffs of using a virtual DOM
 
-## Overview
-
-We want to give students a very high-level look at virtual DOMs in the abstract.
-It's totally okay to focus on React, but students should understand that React's
-isn't the only virtual DOM implementation in town, and they should also
-understand that React is more than simply a virtual DOM (even if that's one of
-its core ideas that it happens to execute very well).
-
-It's worth mentioning some of the tradeoffs w/r/t to reconciling virtual and
-actual state. Inputs are a bit wonky (but more on that in a later lesson, of
-course), and side effects in when rendering can grind an application to a halt.
-
-At the same time, JavaScript is fast while the DOM APIs are slow — this point is
-worth emphasizing.
+## What's a virtual DOM?
+![React VR?](https://media.giphy.com/media/3o7qDL7l1IZAQvxvj2/giphy.gif)
+
+By now, you should already know what the DOM is: a programmatic representation of the document we see in the browser.
+So, what's all this virtual DOM ruckus about? Well, it turns out that our good 'ol friend DOM is slow — terribly slow. He
+makes a granny using a walker look like Usain Bolt.
```

Comment:
Just a gentle, sincere question: might it be wise to avoid the granny reference? Another comparison?